cfgov/unprocessed/apps/owning-a-home/js/explore-rates/RCChart.js

```diff
@@ -0,0 +1,206 @@
+import {
+  chartTooltipMultiple,
+  chartTooltipSingle
+} from './template-loader';
+import Highcharts from 'highcharts';
+import highchartsExport from 'highcharts/modules/exporting';
+import { applyThemeTo } from './highcharts-theme';
+
+function RCChart() {
```

Comment:
Please spell this out.

Reply:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/4045/commits/c081a0227e3718b6ef4edba62461ecf2fdfcefe6